src/operations/cropAlpha.ts

```diff
@@ -0,0 +1,99 @@
+import { IJS } from '../IJS';
+import checkProcessable from '../utils/checkProcessable';
+
+export interface CropAlphaOptions {
+  /**
+   * Threshold above which rows and columns should be kept.
```

Comment:
```suggestion
   * Threshold from which rows and columns should be kept.
```

The default is maxValue and there is nothing above it.